packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -2380,7 +2633,151 @@ test.serial('writeGranuleFromApi() saves granule records to Dynamo, Postgres and
   t.is(postgresRecord.timestamp.getTime(), esRecord.timestamp);
 });
 
-test.serial('writeGranuleFromApi() saves granule records to Dynamo, Postgres and ElasticSearch with same default time values.', async (t) => {
+test.serial('writeGranuleFromApi() saves updated values for running granule record to Dynamo, Postgres and ElasticSearch on rewrite', async (t) => {
```

Comment:
For these tests that change the status of the granule, is it not important to assert that the status of the updated granules are what we expect?

Reply:
That's a very good point - explicit over implicit.  Good catch.

Reply:
f1bb56ecf6